driving_gridworld/obstacles.py

```diff
@@ -49,3 +51,22 @@ def expected_reward_for_collision(self, speed):
 
     def __str__(self):
         return 'p'
+
+
+class CarObstacle(Obstacle):
+    def __init__(self, row, col, prob_of_appearing=0.2, obst_speed=1):
+        self.row = row
+        self.col = col
+        self.prob_of_appearing = prob_of_appearing
+        self.obst_speed = obst_speed
+
+    def expected_reward_for_collision(self, speed):
+        return -8e2**(speed + self.obst_speed)
+
+    def reward_for_collision(self, speed, stddev=0.0):
+        proportional_stddev = stddev * speed
+        return (self.expected_reward_for_collision(speed) +
+                np.random.normal(0, math.sqrt(2)*proportional_stddev))
```

Comment:
I forgot that we're multiplying the supplied standard deviation by speed, so the standard deviation from each car will actually be different, so could you use the full square root of the sum of the variances instead of assuming the variances are the same, please? You also don't need to import `math`, you can just use `np.sqrt` and `np.square`.